File: 5622.java
import java.util.Scanner;

public class Main{
    public static void main(String[] args){
        Scanner sc = new Scanner(System.in);
        int[] inputNum = new int[15];
        int sum = 0;
        String inputString = sc.nextLine();
        
        for(int i = 0; i < 15; i ++)
            inputNum[i] = 0;
        
        for(int i = 0; i < inputString.length(); i++){
            switch(inputString.charAt(i)){
                case 'A': case 'B': case 'C':
                    sum  = sum + 3;
                    break;
                case 'D': case 'E': case 'F':
                    sum  = sum + 4;
                    break;
                case 'G': case 'H': case 'I':
                    sum  = sum + 5;
                    break;
                case 'J': case 'K': case 'L':
                    sum  = sum + 6;
                    break;
                case 'M': case 'N': case 'O':
                    sum  = sum + 7;
                    break;
                case 'P': case 'Q': case 'R': case 'S':
                    sum  = sum + 8;
                    break;
                case 'T': case 'U': case 'V':
                    sum  = sum + 9;
                    break;
                case 'W': case 'X': case 'Y': case 'Z':
                    sum  = sum + 10;
                    break;
            }
        }    
        System.out.println(sum);
    }
}

File: 5622.py
inputString = str(input())
sum = 0

for each in inputString:
    if each == 'A' or each == 'B' or each == 'C':
        sum = sum + 3
    elif each == 'D' or each == 'E' or each == 'F':
        sum = sum + 4
    elif each == 'G' or each == 'H' or each == 'I':
        sum = sum + 5
    elif each == 'J' or each == 'K' or each == 'L':
        sum = sum + 6
    elif each == 'M' or each == 'N' or each == 'O':
        sum = sum + 7
    elif each == 'P' or each == 'Q' or each == 'R' or each == 'S':
        sum = sum + 8
    elif each == 'T' or each == 'U' or each == 'V':
        sum = sum + 9
    else:
        sum = sum + 10

print(sum)
